include/revng/Pipeline/Step.h

```diff
@@ -137,6 +138,12 @@ class Step {
     return llvm::Error::success();
   }
 
+  void setComponent(const llvm::StringRef Component) {
+    this->Component = Component.str();
+  }
```

Comment:
Can we drop this?

Reply:
Do you mean integrating it in the constructor? Dunno, the component is not an *essential* part of a `Step` (e.g. `begin` does not belong to any component)

Reply:
Empty string?

Reply:
> Empty string in such case?

That's the way it is currently

Reply:
Then drop `setComponent` and let's go through the constructor only?

Reply:
Done